packages/web/src/js/Collapse.ts

```diff
@@ -59,14 +61,32 @@ class Collapse extends BaseComponent {
     return `${this.NAME}`;
   }
 
+  siblingsControlHandler(trigger: HTMLElement) {
+    if (this.target?.dataset.parent) {
+      const parentElement = selectorEngine.findOne(`#${this.target?.dataset.parent}`);
+      const children = parentElement?.children;
+
+      if (!children) return;
```

Comment:
```suggestion
      if (!children) {
        return;
      }
```

Please, prefer this syntax for better readability as we do it everywhere.

Reply:
fixed